csharp/Adapter/Microsoft.Spark.CSharp/Sql/DataFrame.cs

```diff
@@ -149,7 +148,7 @@ public DataFrame Select(string firstColumnName, params string[] otherColumnNames
         ///   df.SelectExpr("colA", "colB as newName", "abs(colC)")
         ///   
         /// </summary>
-        /// <param name="columnNames"></param>
+        /// <param name="columns"></param>
```

Comment:
Good catch @skaarthik - param name was also changed in commit 037d183acb9490f929f7971df5b6754b3908a352. Fixed and repushing.